coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -208,7 +240,6 @@ def from_metadata(cls, doccomment, docstyle_definition,
 
         This function just assembles the documentation comment
         itself, without the markers and indentation.
-
```

Comment:
why this change?

Reply:
I don't know when these changed, amending the commit anyways.